external/js/cothority/src/byzcoin/client-transaction.ts

```diff
@@ -320,6 +323,88 @@ export class Instruction extends Message<Instruction> {
         return this.deriveId(what);
     }
 
+    /**
+     * Return a general-purpose json representation of an instruction that is
+     * contracts aware, ie. knows how to render specific contracts arguments. In
+     * the case a problem occurs (for example the requested contract does not
+     * implement a beautify function), the status property is set to non-zero.
+     *
+     * The JSON has the following structure:
+     *
+     *  {
+     *      status: 0 | 1
+     *      type: "spawn|invoke|delete|UNKNOWN",
```

Comment:
Why `UNKNOWN`? Is this even a possibility?

Reply:
In theory it should never happen

Reply:
Then remove it. If the `Instruction` is neither of the three, then it makes sense to `throw new Error`, as this is something that should not happen.